Clean/Circuit/Provable.lean

```diff
@@ -134,28 +134,6 @@ instance : ProvableType unit where
 
 @[reducible] def fieldVar (F : Type) := field (Expression F)
 
--- Instances for arithmetic operations on field expressions
--- Since field is defined as id, field (Expression F) = Expression F
-variable {F : Type} [Field F]
-
-instance : HSub (field (Expression F)) (field (Expression F)) (field (Expression F)) where
```

Comment:
I didn't remove these, just made them smaller and moved next to existing similar instances